components/signup/Auth/index.js

```diff
@@ -0,0 +1,125 @@
+import React, { useCallback, useState, useEffect, useRef } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Link from 'next/link';
+
+import GoogleLogo from 'assets/svg/btn_google_light_normal_ios.svg';
+import { emailRule } from '~/lib/regex';
+import { getVerifyEmailLoading } from '~/redux/selectors/signup';
+import { landingEmailSelector } from '~/redux/selectors/landing';
+import { verifyEmailRequest } from '~/redux/reducers/signup';
+import { saveEmail as saveLandingEmail } from '~/redux/reducers/landing';
+
+import * as CS from '../common/styles';
+import * as S from './style';
+
+const Index = () => {
+  const dispatch = useDispatch();
+  const [emailTestError, setEmailTestError] = useState(false);
+  const verifyEmailLoading = useSelector(getVerifyEmailLoading);
+  const landingEmail = useSelector(landingEmailSelector);
+  const emailEl = useRef(null);
+  const submitButtonEl = useRef(null);
```

Comment:
```suggestion
  const emailRef = useRef(null);
  const submitButtonRef = useRef(null);
```
ref 변수 네이밍은 El보단 Ref 를 명시해주는 편입니다.

Reply:
해당 useRef 의 타입이 DOMElement 가 아닌 string 이기떄문에 더욱 맞는거 같습니다.